### PR TITLE
Minor fixes and updates.

### PR DESCRIPTION
**Pull request by realkot**
Made some tests and changes, so it works like a charm under Ubuntu 18.04 LTS without any errors and already contains changes from this PR: https://github.com/criegerde/install-nextcloud/pull/18

Log from test run: https://www.dropbox.com/s/y2nz5jaaramsbfo/putty.log?dl=0

@Akito13 can u take a look?

### CODE REVIEW

File: install-nextcloud-mariadb-ubuntu.sh [hunk 1]
```diff
@@ -33,23 +33,38 @@ fail2ban-client status nextcloud
...
+apt install -y software-properties-common zip unzip screen git ffmpeg libfile-fcntllock-perl locate ghostscript tree htop
+
 ###instal NGINX using TLSv1.3, OpenSSL 1.1.1
-update_and_clean
-apt install nginx -y
+###we have to install nginx-extras for workaround with:
+### nginx[23284]: nginx: [emerg] unknown directive "mp4" in /etc/nginx/conf.d/nextcloud.conf:48
+#apt install -y nginx
+#apt install -y nginx-full
+apt install -y nginx-extras
```
Comment (Akito13) [inline install-nextcloud-mariadb-ubuntu.sh]: I just checked that `nginx-extras` only depends on `nginx-common`. Maybe add `nginx-full` or uncomment line 65?

File: README.md [hunk 2]
```diff
@@ -20,3 +20,22 @@ The scripts called install-nextcloud-<database>-debian.sh and install-nextcloud-
 Ready to go (?) … let’s start.
 
 Carsten Rieger: https://www.c-rieger.de
+
+
+# Short example for Ubuntu 18.04 LTS
```
Comment (Akito13) [inline README.md]: I personally would probably add _Copy and paste the following commands into the terminal, line by line:_ as a subtitle/description and remove line 26 + 33, to make it clearer.

File: install-nextcloud-mariadb-ubuntu.sh (hunk 1 above)
Comment (realkot) [reply install-nextcloud-mariadb-ubuntu.sh]: > I just checked that `nginx-extras` only depends on `nginx-common`. Maybe add `nginx-full` or uncomment line 65?

Not needed.
https://askubuntu.com/questions/553937/what-is-the-difference-between-the-core-full-extras-and-light-packages-for-ngi:
image

Comment (Akito13) [reply install-nextcloud-mariadb-ubuntu.sh]: @realkot
Nice to know, never heard of nginx-extras before.

**Comment by Akito13** [comment]
👍

File: README.md (hunk 2 above)
Comment (realkot) [reply README.md]: Done

**Comment by criegerde** [comment]
i switch from odrejs repository to the origin nginx one. That's why i commented this line (
_#apt install -y nginx-full_) out. It isn't necessary to install _apt install -y nginx-extras_ anymore. 
If nginx would fail caused by the mp4 directive, something went wrong earlier. May be it is an unsopprted architecture (e.g. RPi4 / ARM64...)? Check _nginx -v_ and you will see, it isn't the latest release.
Nevertheless, thank you - i will have a look.
Cheers, Carsten

**Pull request closed without merging**